File: Array/26-Remove Duplicates from Sorted Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        if len(nums)==0:
            return 0
        cur = 0
        for i in range(1, len(nums)):
            if nums[i] != nums[cur]:
                cur += 1
                nums[cur] = nums[i]
        return cur+1

File: Array/26-Remove Duplicates from Sorted Array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        if (nums.Length == 0) return 0;
        int cur = 0;
        for (int i = 1; i < nums.Length; i++){
            if (nums[i] != nums[cur]){
                cur++;
                nums[cur] = nums[i];
            }
        }
        return cur + 1;
    }
}